.github/pull_request_template.md

```diff
@@ -1,69 +1,78 @@
-- [ ] The code is composed of functions of no more than 50 lines and uses meaningful names for variables and follows the [style guide](https://docs.esmvaltool.org/en/latest/community/introduction.html#code-style)
-- [ ] [Documentation](https://docs.esmvaltool.org/en/latest/community/introduction.html#documentation) is available
-- [ ] Please use `yamllint` to check that your YAML files do not contain mistakes
-- [ ] (Only if really necessary) Add any additional dependencies needed for the diagnostic script to setup.py, esmvaltool/install/R/r_requirements.txt or esmvaltool/install/Julia/Project.toml (depending on the language of your script) and also to package/meta.yaml for conda dependencies (includes Python and others, but not R/Julia). Also check that the license of the dependency you want to add and any of its dependencies are compatible with [Apache2.0](https://github.com/ESMValGroup/ESMValTool/blob/master/LICENSE).
+* * *
 
-New or updated [recipe/diagnostic](https://docs.esmvaltool.org/en/latest/community/diagnostic.html):
+## Before you get started
 
-- [ ] Documentation for the recipe/diagnostic is available in the `doc/sphinx/source/recipes` folder and an entry has been added to `index.rst`
-- [ ] [Provenance information](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recording-provenance) has been added and no warnings related to provenance are generated when running the recipe
+<!--
+    Please discuss your idea with the development team before getting started,
+    to avoid disappointment or unnecessary work later. The way to do this is
+    to open a new issue on GitHub.
+-->
 
-New or updated [data reformatting script](https://docs.esmvaltool.org/en/latest/develop/dataset.html):
+- [ ] [☝ Create an issue](https://github.com/ESMValGroup/ESMValTool/issues) to discuss what you are going to do
 
-- [ ] Add a new dataset to the table in the [documentation](https://docs.esmvaltool.org/en/latest/input.html#supported-datasets)
-- [ ] Add a test for the CMORized data to recipes/example/recipe_check_obs.yml and run the recipe, to make sure the CMOR checks pass without errors
-- [ ] There are clear instructions on how to obtain the data
-- [ ] Tag @remi-kazeroni in this pull request, so that the new dataset can be added to the OBS data pool at DKRZ and synchronized with CEDA-Jasmin
+## Checklist
 
-Automated checks pass, status can be seen below the pull request:
+It is the responsibility of the author to make sure the PR is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.
 
-- [ ] Circle/CI tests pass. If the tests are failing, click the `Details` link to find out why.
-- [ ] Preferably Codacy code quality checks pass, however a few remaining hard to solve Codacy issues are still acceptable. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
```

Comment:
It looks like the text about a few issues remaining went missing, could you add it somewhere again? Maybe in the checklist?

Reply:
Added to the contributing guidelines